lib/pf/services/manager/collectd.pm

```diff
@@ -58,7 +61,10 @@ sub generateCollectd {
     $tags{'graphite_host'} = "$Config{'monitoring'}{'graphite_host'}";
     $tags{'graphite_port'} = "$Config{'monitoring'}{'graphite_port'}";
     $tags{'hostname'}      = hostname;
-    $tags{'db_host'}       = "$Config{'database'}{'host'}";
+    $tags{'db_host'}
+        = $cluster_enabled
+        ? $management_network->tag('ip')
```

Comment:
Are you sure ? https://github.com/inverse-inc/packetfence/blob/devel/docs/PacketFence_Clustering_Guide.asciidoc#mysql-network-configuration